### ISSUE TITLE
[MU4 Task] Connect our Keyboard Navigation system to QWidget Dialogs

### ISSUE THREAD

**Issue by cbjeukendrup** [musescore/MuseScore#9479]
**Task description**
Currently, keyboard navigation does not work at all in QWidget Dialogs; neither our own system nor QWidget's default system. We need to somehow connect our system to the QWidget dialogs, to make sure both systems work seamlessly together.
